### DISCUSSION TITLE
[Feature Request]: 洗版功能建议

### DISCUSSION THREAD

**Discussion by iceucn** [jxxghp/MoviePilot#3470] [讨论]
### 当前程序版本

v1.9.2

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

洗版问题与建议：
电影洗版能否在订阅的时候2个选择：
    1：订阅（直接下载整理入库）；
    2：订阅+洗版（先下载入库，如果非洗版命中，继续按时执行洗版规则进行洗版）；
    
剧集洗版在订阅的时候3个选择：
    1：订阅（直接下载整理入库）；
    2：订阅+洗版（同电影，按集洗版）；
    3：完结洗版（剧集完结后，执行洗版任务，根据洗版规则，找剧集的单种子合集包下载）

历史记录单独记录洗版记录，目前洗版是洗版文件与原文件共存还是？如果共存的话，洗版整理记录里面能批量删除非洗版文件，只保留最终洗版文件内容

**Comment by libo26** [comment]
非常期待按集洗版的功能

**Reply by zzstar101** [reply]
这个功能我觉得很有必要

**Comment by zzstar101** [comment]
期望作者开发按集洗版的功能

**Comment by alin-alin-alin** [comment]
期望作者开发按集洗版的功能
